arborist/auth.go

```diff
@@ -548,6 +548,7 @@ func authorizedResources(db *sqlx.DB, request *AuthRequest) ([]ResourceFromQuery
 			LoggedInGroup,    // $3
 		)
 		if err != nil {
+			fmt.Println(err.Error())
```

Comment:
Ugh maybe we shouldn't log like this here, was just being lazy, but I believe a better solution would be to log the actual error in `response.go` on `line 82`? instead of 
```
response.log.Error(message)
```

Maybe we should have the message and the error logged? Would also probably cover most errors we log using `newErrorResponse` in arborist.